ds-prep-python-sets/main.py

```diff
@@ -0,0 +1,48 @@
+# 3
+trail_mix = set({'m&ms', 'peanuts', 'raisins'})
+
+# 4
+print(trail_mix)
+
+# 6
+print('cashews is in trail_mix: {}'.format('cashew' in trail_mix))
+
+# 7
+print('peanuts is in trail_mix: {}'.format('peanuts' in trail_mix))
+
+# 9
+trail_mix.add('pretzels')
+
+# 10
+trail_mix.update(['peanuts', 'banana chips', 'bits of jerky'])
+
+# 11
+print("Original set trail_mix after adding 'pretzels', 'peanuts', 'banana chips', and 'bit of jerky': {}".format(trail_mix))
+
+# 13
+trail_mix.remove('m&ms')
+
+# 14
+trail_mix.discard('raisins')
+
+# 15
+trail_mix.discard('rat poison')
+
+# 16
+print("Set trail_mix after removing 'm&ms', 'raisins', and 'rat poison': {}".format(trail_mix))
+
+# 17
+nuts = set({'peanuts', 'cashews', 'almonds', 'walnuts', 'pecans'})
```

Comment:
As above, the `set` function call is unnecessary here.

---

![image](https://user-images.githubusercontent.com/16143771/148819460-d7a0383e-2c76-44ef-b039-76e520283ab7.png)